ios/BetterRailWidget/Live Activity/Views/ActivityHeader.swift

```diff
@@ -4,14 +4,29 @@ struct ActivityHeader: View {
   var vm: ActivityViewModel
   
   var body: some View {
-    HStack(spacing: vm.isRTL ? 4 : 6) {
-      Image(systemName: "train.side.front.car")
-        .scaleEffect(x: vm.isRTL ? -1 : 1, y: 1, anchor: .center)
+    HStack {
+      HStack(spacing: vm.isRTL ? 4 : 6) {
+        Image(systemName: "train.side.front.car")
+          .scaleEffect(x: vm.isRTL ? -1 : 1, y: 1, anchor: .center)
+        
+        Text("BETTER RAIL")
+      }
+      .font(.caption2)
+      .foregroundColor(Color(uiColor: .systemGray))
+      .padding(.bottom, -4)
       
-      Text("BETTER RAIL")
+      Spacer()
+        
+      if (vm.isStale) {
+        HStack(spacing: 4.0) {
+          Image(systemName: "exclamationmark.circle.fill")
+            .font(.caption2)
+          
+          Text("connection issues")
+            .font(.caption2)
+        }
+        .foregroundColor(.orange).fontWeight(.heavy)
```

Comment:
Changing to red might be more appropriate:

![IMG_6179](https://github.com/better-rail/app/assets/12946462/5c520017-2c3b-44b7-b891-f22f2bd464c2)

Reply:
it'll be mixed up with the delay color :/